flatdata-cpp/include/flatdata/MultiVector.h

```diff
@@ -57,54 +76,61 @@ class MultiVector
     ExternalVector< IndexType > m_index;
 
     std::unique_ptr< ResourceHandle > m_handle;
-    size_t m_size = 0;
+    size_t m_size_serialized = 0;
```

Comment:
`m_bytes_flushed` to be even more precise? Otherwise this managed to confuse me a bit at line 142

Reply:
Done